engine/class_modules/sc_shaman.cpp

```diff
@@ -4087,6 +4095,16 @@ struct lightning_bolt_t : public shaman_spell_t
     }
   }
 
+  double composite_maelstrom_gain_coefficient( const action_state_t* state ) const override
+  {
+    double coeff = shaman_spell_t::composite_maelstrom_gain_coefficient( state );
+
+    if ( p()->conduit.high_voltage->ok() && rng().roll( p()->conduit.high_voltage.percent() ) )
+      coeff *= 2.0;
```

Comment:
We need to make sure to see if generation bug is fixed in beta or when shadowlands launches.